core/templates/dev/head/pages/exploration_player/AnswerFeedbackPairDirective.js

```diff
@@ -66,6 +66,10 @@ oppia.directive('answerFeedbackPair', [
             }
             return shortAnswerHtml;
           };
+
+          $scope.getRequestForHint = function() {
+            return $scope.data.learnerAnswer;
```

Comment:
This is not what I meant. Asking for a hint is not the same as submitting an answer, and conflating them just makes things muddy.

Try to find a way to pass in (via $scope.data or some additional boolean arg) whether this is a hint request or not. Use ng-if="..." in the HTML to determine whether to show the request/hint pair instead of the usual answer/feedback pair.